Doc/library/atexit.rst

```diff
@@ -83,21 +83,22 @@ making an explicit call into this module at termination. ::
        _count = _count + n
 
    def savecounter():
-       with open("counterfile", "w") as outfile:
-           outfile.write("%d" % _count)
+       with open('counterfile', 'w') as outfile:
+           outfile.write('%d' % _count)
 
    import atexit
+
    atexit.register(savecounter)
 
 Positional and keyword arguments may also be passed to :func:`register` to be
 passed along to the registered function when it is called::
 
    def goodbye(name, adjective):
```

Comment:
shouldn't these be kwargs for the example below to work?

Reply:
@iritkatriel I have just tried it on Python 3.9.2 and both positional and keyword arguments work.